### DISCUSSION TITLE
Auto backups doesn't work?

### DISCUSSION THREAD

**Discussion by NikitaSharapov** [pockethost/pockethost#390] [Q&A] [answered]
Automatic backup does not work, I assume this is due to "PocketHost runs your PocketBase instance on-demand. That means PocketHost waits for a web request to hit PocketBase before it actually launches PocketBase and responds.". It turns out that my automatic backups will not work if pocketbase instance is not running at this point in time?

And another question. Will access to the backups you make automatically be released? for example, recovery points via the interface app.pockethost.io/app/instances ?

**Comment by benallfree** [comment, marked as answer]
Yes, automatic backups only work when pocketbase is running. We are working on some ideas to improve that. 

Backup history is kept according to the rules you set in the pocketbase admin.

**Reply by NikitaSharapov** [reply]
Thank you very much for the answer, I am glad to hear that you are already working on it.
As for access to backups, after reading this (https://pockethost.io/docs/overview/faq/#how-often-does-my-data-get-backed-up) in the documentation, I thought that you also make a backup of my PB (I don't know if it is a container or a volume or some other magic) independently of my manual backups from the PB admin panel

**Reply by benallfree** [reply]
The backups mentioned in the docs are daily server backups, they aren't accessible to individual users and instances.
